CONTRIBUTING.md

```diff
@@ -139,7 +137,7 @@ included in the project:
     with a clear title and description against the `master` branch.
 
 **IMPORTANT**: By submitting a patch, you agree to allow the project owners to
-license your work under the terms of the [BSD License](LICENSE).
+license your work under the terms of the [BSD License](LICENSE.md).
```

Comment:
It's actually BSD, I changed the LICENSE file below by mistake but I'm changing it back